jobs/build/ocp/Jenkinsfile

```diff
@@ -81,6 +101,9 @@ ${OSE_CHANGELOG}
 
 ===OpenShift Ansible changelog snippet===
 ${OA_CHANGELOG}
+
+${img_fail_msg}
```

Comment:
Let's move this information up above the changelogs. Particularly information about failing images needs to be very visible.